probe_scraper/transform_probes.py

```diff
@@ -172,19 +170,18 @@ def transform(probe_data, node_data, break_by_channel):
            release channel.
     """
     channels = sorted_node_lists_by_channel(node_data)
-
-    result_data = {}
+    result_data =defaultdict(dict)
     for channel, channel_data in channels.iteritems():
-        print "\n" + channel + " - transforming probe data:"
+        print "\n" + str(channel) + " - transforming probe data:"
```

Comment:
Why did you change this part?

Reply:
I have just change it. Not any Reason behind it.

Reply:
In general, if there isn't any specific reason to change a line, then it's better to leave it alone :) That makes it less likely to add new bugs to the code!